lineman/app/components/memberships_page/admin_memberships_panel/admin_memberships_panel_controller.coffee

```diff
@@ -9,6 +9,9 @@ angular.module('loomioApp').controller 'AdminMembershipsPanelController', ($scop
 
   $scope.toggleAdmin = (membership) ->
     method = if membership.admin then 'makeAdmin' else 'removeAdmin'
+    if !membership.admin and membership.user() == CurrentUser and !confirm($filter('translate')('memberships_page.remove_admin_from_self.question'))
```

Comment:
If we decide to go the confirm route (I'm having second thoughts about it as I type), we'll want to separate this out into a service of some kind so we can do something nice like

```
if UserConfirmation.confirm('memberships_page.remove_admin.question')
  // do stuff
```

But this is good for now.